README.md

```diff
@@ -52,6 +57,23 @@ For information about the API, see the [API documentation](API.md).
 
 For information about testing the service, see the [Testing documentation](TESTING.md).
 
+## Secrets
+
+Secrets for this project follow the naming convention `${EnvironmentType}/${DNSRecord}/honeybadger_api_key`. Our security model controls access to secrets based on the prefix, and will deny read access to any secret not prefixed with `development/`, so we tell our application's Cloudformation template whether we are in a development, test, or production environment type via [config files](cicd/3-app/aiproxy/config).
+
+* In production: `production/aiproxy.code.org/honeybadger_api_key`
```

Comment:
Hey, I'm still learning about how we handle open-source projects. This one looks like anyone could use it, but it has some code.org stuff in it.

If someone else wants to use or help with AIProxy, the code.org references might be confusing. Maybe we could explain in the README that this started at code.org but can be used by anyone? This might make things clear for everyone.  Or, alternatively we could be clear that is this part of our stack, please fork if you want to use it.

Reply:
It is a goal for this service (and it does so now) to be able to stand up on its own with a local configuration via docker. This is for local development without needing AWS access and for external contribution and/or use. We could clarify that specifically in the README.

Reply:
For the main `code-dot-org` repository, we use a config file which can be populated all with your own values, but if you populate it with `!Secret` then we will attempt to fetch it from AWS. We could follow a similar pattern here.

That said, this secret is created _by_ this project. The ci/cd is specifically for AWS, but if you're deploying to AWS, there's nothing specifically code.org that you need. Now the various `!ImportValue` references to external resources are a different story...